.env.staging

```diff
@@ -0,0 +1,12 @@
+# Proxy urls
+VITE_USE_PROXYURL=true
+VITE_PROXYURL_WMS="https://map.geoportail.lu/ogcproxywms"
+VITE_PROXYURL_REMOTE="https://map.geoportail.lu/httpsproxy"
+
+# Urls for vectortiles
+VITE_VECTORTILES_URL="https://vectortiles-staging.geoportail.lu"
+
+# Paths for MVT styles
+VITE_MVTSTYLES_PATH_GET="https://migration.geoportail.lu/getvtstyle"
+VITE_MVTSTYLES_PATH_UPLOAD="https://migration.geoportail.lu/uploadvtstyle"
+VITE_MVTSTYLES_PATH_DELETE="https://migration.geoportail.lu/deletevtstyle"
```

Comment:
On local dev in v3 I get CORS errors for these. I'm wondering 
1. if we should rather keep the relative paths like `"/getvtstyle"` and introduce a separate mode for gh-pages
2. or just use a CORS browser plugin on local dev

For option 1. I'm not sure where the calls to `localhost:8080/uploadvtstyle` actually end up. Do they stay locally, or are they forwarded to migration or prod somehow?

Reply:
> 
> For option 1. I'm not sure where the calls to `localhost:8080/uploadvtstyle` actually end up. Do they stay locally, or are they forwarded to migration or prod somehow?

Looks like it ends up to a 405, see gh-pages pointing on `main` : 
https://geoportail-luxembourg.github.io/luxembourg-geoportail/main

Reply:
local calls are forwarded to `backendHost` which is defined as `const backendHost = 'http://localhost:8080'`

 https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/blob/GSLUX-698-fix-mvt-urls/vite.config.ts#L40C24-L40C35

so if you have a v3 instance running locally listening on port 8080, it will be used